extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -118,11 +121,31 @@ class AmpFitText extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    if (this.win.ResizeObserver) {
```

Comment:
please flip to save a level of indentation:
```js
if (!this.win.ResizeObserver) {
  return;
}
```

Reply:
Would it make more sense to just pull this part into the same if check as the following one? like 
```suggestion
    if (this.win.ResizeObserver && this.resizeObserverUnlistener_ === null) {
```

Reply:
You still want to early return to save a level of indentation.
```js
if (!this.win.ResizeObserver || this.resizeObserverUnlistener_) {
  return;
}
```

Reply:
In this case we want to call the final return statement whether or not this if block is true, this specifically relates to deciding whether we should attach a new observer, would it be better to extract the logic which attaches the listener to a helper method? so that the layout callback would look like this?
`if (this.win.ResizeObserver && this.resizeObserverUnlistener_ === null) {
    attachNewResizeObserver();
}
return this.mutateElement(() => {
   this.updateFontSize_();
});`

Reply:
Oh I see, apologies for missing that. In that case either way works :)

Reply:
Sounds good, Thanks!